src/main/java/sirius/biz/analytics/events/EventRecorder.java

```diff
@@ -539,4 +542,17 @@ protected Event<?> fetchBufferedEvent() {
 
         return result;
     }
+
+    /**
+     * Fetches all events which match the given query considering the given duplicate preventer.
```

Comment:
I don't find it very clear in the API what the duplicate prevent should be or what it can or shouldn't do. Could this be clarified in the Docs?